<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>formYeniBaglanti</class>
 <widget class="QDialog" name="formYeniBaglanti" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Müşteri Penceresi</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="simgeler.qrc" >:/new/prefix1/icons/identity.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="lineTelefon" />
     </item>
     <item row="6" column="1" >
      <widget class="QTextEdit" name="textAdres" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboSektor" />
       </item>
       <item>
        <widget class="QPushButton" name="buttonEkle" >
         <property name="text" >
          <string>« &amp;Ekle</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineSektor" />
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Sektör</string>
       </property>
       <property name="buddy" >
        <cstring>comboSektor</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;E-Posta</string>
       </property>
       <property name="buddy" >
        <cstring>lineEposta</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="lineEposta" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Müşteri Adı</string>
       </property>
       <property name="buddy" >
        <cstring>lineBaglanti</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>&amp;Adres</string>
         </property>
         <property name="buddy" >
          <cstring>textAdres</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="lineVergiNo" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>&amp;Telefon</string>
       </property>
       <property name="buddy" >
        <cstring>lineTelefon</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineBaglanti" />
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="lineVergiDairesi" />
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>&amp;Vergi No</string>
       </property>
       <property name="buddy" >
        <cstring>lineVergiNo</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Vergi &amp;Dairesi</string>
       </property>
       <property name="buddy" >
        <cstring>lineVergiDairesi</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;Kaydet</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>İ&amp;ptal</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>lineBaglanti</tabstop>
  <tabstop>comboSektor</tabstop>
  <tabstop>buttonEkle</tabstop>
  <tabstop>lineSektor</tabstop>
  <tabstop>lineEposta</tabstop>
  <tabstop>lineTelefon</tabstop>
  <tabstop>lineVergiNo</tabstop>
  <tabstop>lineVergiDairesi</tabstop>
  <tabstop>textAdres</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="simgeler.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>formYeniBaglanti</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
